staging/src/k8s.io/client-go/tools/cache/fifo.go

```diff
@@ -84,8 +84,9 @@ type Queue interface {
 // unless you really really really really know what you are doing.
 func Pop(queue Queue) interface{} {
 	var result interface{}
-	queue.Pop(func(obj interface{}) error {
+	queue.Pop(func(obj interface{}, isInInitialList bool) error {
 		result = obj
+		// TODO: handle isInInitialList?
```

Comment:
nit: either clarify what type of handling this is asking for or drop the todo

Reply:
Fixed (and marked this function for deprecation)